src/platform/packages/shared/kbn-dom-drag-drop/src/sass/drag_drop_mixins.scss

```diff
@@ -40,17 +40,17 @@ $domDragDropZLevel3: 3;
 
 // Style for drop area when there's an item being dragged
 @mixin mixinDomDroppableActive($borderWidth: $euiBorderWidthThin) {
-  background-color: transparentize($euiColorVis0, .9) !important;
+  background-color: transparentize($euiColorSuccess, .9) !important;
   &:before {
-    border-color: $euiColorVis0 !important;
+    border-color: $euiColorSuccess !important;
```

Comment:
I believe this variable should be changed to `$euiColorBorderBaseSuccess`. Please note that this variable may be bugged/erroneous at the moment, but the EUI team is working on a PR to remedy it.

Reply:
done in [49d8096](https://github.com/elastic/kibana/pull/204839/commits/49d8096ab969b7f4b1a834b4ebba60b373c80808)